snap/epoch.go

```diff
@@ -159,6 +159,25 @@ func (e *Epoch) IsZero() bool {
 	return rZero && wZero
 }
 
+func ueq(a, b []uint32) bool {
```

Comment:
I would maybe call this epochListEq instead, or something similar?

Reply:
done